modules/preview-react/information-highlight/lib/Icon.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {SystemIcon, SystemIconProps} from '@workday/canvas-kit-react/icon';
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {cssVar} from '@workday/canvas-kit-styling';
+import {
+  infoIcon,
+  exclamationCircleIcon,
+  exclamationTriangleIcon,
+} from '@workday/canvas-system-icons-web';
+import {base} from '@workday/canvas-tokens-web';
+import {useInformationHighlightModel} from './modelHook';
+
+type Variant = 'informational' | 'caution' | 'attention';
+
+const iconStyles = {
+  informational: {
+    accent: cssVar(base.frenchVanilla100),
+    background: cssVar(base.blueberry400),
+    color: cssVar(base.blueberry400),
+  },
+  caution: {
+    accent: cssVar(base.frenchVanilla100),
+    background: cssVar(base.cantaloupe400),
+    color: cssVar(base.cantaloupe400),
+  },
+  attention: {
+    accent: cssVar(base.frenchVanilla100),
+    background: cssVar(base.cinnamon500),
+    color: cssVar(base.cinnamon500),
+  },
+};
+
+export interface IconProps extends SystemIconProps {
```

Comment:
I'd probably put this at the top of the file instead of dividing `iconStyles` and `defaultIcons`, but I don't know canvas' conventions on where exports should go. I suspect not in the middle though :)

Reply:
We usually like to keep our "exports" at the top.